migration/mig_0016_add_recommendations_table.go

```diff
@@ -31,19 +31,18 @@ var mig0016AddRecommendationsTable = Migration{
 					error_key
 				FROM rule_hit;
 			`)
-			if err != nil {
-				return err
-			}
-			// stop here if sqLite
-			return nil
+
+			// stop here if not working with postgres
+			return err
 		}
 
+		// Create recommendation table using records from rule_hit table
 		_, err := tx.Exec(`
 			CREATE TABLE recommendation
 				AS SELECT
 					org_id,
 					cluster_id,
-					REGEXP_REPLACE(rule_fqdn, 'report$', error_key) AS rule_fqdn,
+					REGEXP_REPLACE(rule_fqdn, '.report$', '') AS rule_fqdn,
```

Comment:
Is it a good idea to modify older migrations that are, probably, already applied to the DB structure? It won't break anything in local deployments, but maybe it can fail when deploying to prod

Reply:
It won't do anything =)
But I thought I removed this change from the PR. Thanks for noticing!